cypress/e2e/sliding-sync/sliding-sync.ts

```diff
@@ -319,4 +319,18 @@ describe("Sliding Sync", () => {
             "Join", "Test Room",
         ]);
     });
+
+    it("should show a favourite DM only in the favourite sublist", () => {
+        cy.createRoom({
+            name: "Favourite DM",
+            is_direct: true,
+        }).as("room").then(roomId => {
+            cy.contains(".mx_RoomSublist", "Apple");
```

Comment:
Do you need actually need this? You surely don't prior to setting the room tag?

Reply:
Heck, I didn't update it from Apple to `Favourite DM` so it definitely wasn't working :P